docs/egui_demo_app.js

```diff
@@ -7,13 +7,40 @@ let wasm_bindgen;
     }
```

Comment:
please revert `.js` and `.wasm` - see https://github.com/emilk/egui/blob/master/CONTRIBUTING.md#making-a-pr

Reply:
I tried to do but some how i commited with that!
I don't know the commit i did that!

I run `git restore docs --source upstream/master` now is ok!